eth/block.go

```diff
@@ -290,6 +307,53 @@ func (b Block) MarshalJSON() ([]byte, error) {
 		}
 
 		return json.Marshal(&e)
+	case "celo":
+		type celo struct {
+			Number           *Quantity       `json:"number"`
+			Hash             *Hash           `json:"hash"`
+			ParentHash       Hash            `json:"parentHash"`
+			LogsBloom        Data256         `json:"logsBloom"`
+			TransactionsRoot Data32          `json:"transactionsRoot"`
+			StateRoot        Data32          `json:"stateRoot"`
+			ReceiptsRoot     Data32          `json:"receiptsRoot"`
+			Miner            Address         `json:"miner"`
+			TotalDifficulty  Quantity        `json:"totalDifficulty"`
+			ExtraData        Data            `json:"extraData"`
+			Size             Quantity        `json:"size"`
+			GasLimit         Quantity        `json:"gasLimit"`
+			GasUsed          Quantity        `json:"gasUsed"`
+			Timestamp        Quantity        `json:"timestamp"`
+			Transactions     []TxOrHash      `json:"transactions"`
+			Randomness       *Randomness     `json:"randomness,omitempty"`
```

Comment:
According to your [above conditions](https://github.com/INFURA/go-ethlibs/pull/72/files#diff-4f907297b7a261d2aa07c67a0a196dc943544cda780dd8f683cdb883d29119b1R79) to decide the `celo` flavor `randomness` is required not to be nil but here you consider it as `omitempty`. is this right somehow?

Reply:
no you're right it's an error, it's not omitempty